ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -87,7 +87,8 @@ tests:
         ci-org-name openshift-kni
     test:
     - as: e2e-aws-ci-tests
-      commands: FEATURES_ENVIRONMENT=ci IPERF3_BITRATE_OVERRIDE="300000000" make functests-on-ci
+      commands: FEATURES_ENVIRONMENT=ci IPERF3_BITRATE_OVERRIDE="300000000" TARGET_RELEASE=release-4.19
```

Comment:
Should master point to master/main in the repos?

Reply:
it can, but setting it to release-4.19 will be (hopefully) useful when we'll branch release-4.20, as a find/replace procedure will update this variable.

Otherwise, the `openshift-kni-cnf-features-deploy-release-4.19.yaml` file will be created with `TARGET_RELEASE=main`

Reply:
sounds good